### PR TITLE
[Workflow] add getter for workflow

### LINKED ISSUE #23695: Please add getter methods for private workflow properties
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.x

Please add getter methods for the following properties:

* Symfony\Component\Workflow\Workflow#markingStore
* Symfony\Component\Workflow\MarkingStore\MultipleStateMarkingStore#property
* Symfony\Component\Workflow\MarkingStore\SingleStateMarkingStore#property
* Symfony\Component\Workflow\SupportStrategy\ClassInstanceSupportStrategy#className

Currently the only (dirty) way to access this information seems using Reflection.

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #23695
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
retargeted for 3.4 as 3.3 is for bugfixes only

**Comment by Simperfit** [comment]
@nicolas-grekas rebase done

File: src/Symfony/Component/Workflow/Workflow.php [hunk 1]
```diff
@@ -186,6 +186,14 @@ public function getDefinition()
         return $this->definition;
     }
 
+    /**
+     * @return MultipleStateMarkingStore
```
Comment (lyrixx) [inline src/Symfony/Component/Workflow/Workflow.php]: `MarkingStoreInterface`

**Review by lyrixx — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Workflow/Workflow.php (hunk 1 above)
Comment (Simperfit) [inline src/Symfony/Component/Workflow/Workflow.php]: done

**Review by lyrixx — APPROVED** [review]

**Comment by lyrixx** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
ping @Nyholm ;)

**Review by Nyholm — APPROVED** [review]
Thank you for the ping. 

Also @Guite, real nice tool =)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Simperfit.

**Pull request merged**